provider/caleta/provider_service_test.go

```diff
@@ -199,7 +218,10 @@ func TestRequestingGameLaunch(t *testing.T) {
 			app := fiber.New()
 			ctx := app.AcquireCtx(&fasthttp.RequestCtx{})
 			defer app.ReleaseCtx(ctx)
-			s, err := NewCaletaService(test.config, mockClient{PostJSONFunc: test.postFn})
+			// api, err := NewAPIClient(mockRestClient{PostJSONFunc: test.postFn}, test.config)
+			// assert.NoError(t, err)
```

Comment:
Could the commented out code in these tests be removed too?

Reply:
Fixed!